### PR TITLE
Added CONTRIBUTING section

### PR DESCRIPTION
**Pull request by bpas247**
- [x] Add Restrictions section mentioned in "Development" project board

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Contribution Guidelines
+
+Any change to this repository is always welcomed, and greatly appreciated. You can add new entries, or modify/remove existing entries if they are changed/discontinued. You are also allowed to add/modify categories as you see fit.
```
Comment (Chris-Johnston) [inline CONTRIBUTING.md]: I would reword this first sentence and remove information that is redundant.

"Contributions to the list are greatly appreciated and always welcome. Feel free to add and reorganize items as you see fit for a Pull Request"

Though that being said, I think we should avoid giving people the impression that they should reorganize the entire list with each PR.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Please ensure your pull request adheres to the following guidelines:
+
+-   Search previous suggestions before making a new one, as yours may be a duplicate.
```
Comment (Chris-Johnston) [inline CONTRIBUTING.md]: This may be a non-issue, the `duplicate` label exists for a reason?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+-   Make an individual pull request for each suggestion.
+-   If adding/modifying an entry, use the following format:
+    `[NAME](LINK) - DESCRIPTION.`
```
Comment (Chris-Johnston) [inline CONTRIBUTING.md]: Instead of proposing which format to follow for new items, I would instead say something along the lines of "keep a consistent format with the rest of the document". If the document style changes, new entries will be entered differently as well.

Comment (bpas247) [reply CONTRIBUTING.md]: I honestly doubt this format would change, especially if we decide to adhere to the same format of awesome lists. What I could do is re-word is so it sounds more like a suggestion than a strict guideline

File: CONTRIBUTING.md (hunk 2 above)
Comment (bpas247) [reply CONTRIBUTING.md]: I think it is still useful to specify that one should check for duplicates before submitting a PR, so we don't have to check it ourselves. Although it does seem a bit redundant

File: CONTRIBUTING.md (hunk 1 above)
Comment (bpas247) [reply CONTRIBUTING.md]: Maybe something like:

> Contributions to the list are greatly appreciated and always welcome. Feel free to add and reorganize items and categories as you see fit. Although, it is not necessary to reorganize every item with every Pull Request.

Comment (Chris-Johnston) [reply CONTRIBUTING.md]: I think just the first two sentences are good enough.

File: CONTRIBUTING.md (hunk 2 above)
Comment (Chris-Johnston) [reply CONTRIBUTING.md]: ok, that sounds fine
